app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.java

```diff
@@ -32,22 +32,25 @@
 				"bureau_de_change",	"money_transfer", "post_office", "library",
 				"courthouse", "embassy", "car_wash", "car_rental",
 				"marketplace", "fuel", "driving_school", "dentist",
-				"doctors", "clinic", "pharmacy", "veterinary" };
+				"doctors", "clinic", "pharmacy", "veterinary",
+				"place_of_worship", "school", "townhall"};
```

Comment:
Alright, I understand your point of view.
I was disagreeing with the 'very private' and 'not useful' but your point stands certainly for number of contributions/false positives.

Maybe in the future, for yet another quest with lower priority... public-transport-wheelchair > business-wheelchair > school-wheelchair